_issues/wearable-tools-tizenstudio.md

```diff
@@ -1,34 +1,31 @@
----
-title:  "Tizen Baseline SDK"
-permalink: /issues/wearable/tizen-baseline-sdk/
-search: true
-toc: true
-toc_sticky: true
-toc_label: Known Issues & Workarounds
----
-
-
-## Connection issue between a watch and Device Manager
-
-Refer to [Testing Your App on Galaxy Watch][testing_app_on_watch] for information about developing your application and testing it on your watch.
-
-Even if you follow every step correctly, you may face the following error:
-
-![][device_mgr_connection]
-
-```
- ERROR: failed to connect to remote target '192.168.0.X'
-
- You may get this message for following reasons:
-  - This remote device is already connected by another one.
-  - This remote device is running on a non-standard port.
-  - There is no IP address, please check the physical connection.
-```
-
-### Workaround
- - Check whether both devices are connected to the same Wi-Fi network.
- - Reboot your PC and the watch, and connect to the network.
-
-
-[device_mgr_connection]: {{site.url}}{{site.baseurl}}/assets/images/issues/tools/device-manager-connection-issue.png
-[testing_app_on_watch]: https://developer.samsung.com/galaxy-watch/develop/testing-your-app-on-gear
+---
+title:  "Tizen Baseline SDK"
+permalink: /issues/wearable/tizen-baseline-sdk/
+search: true
+toc: true
+toc_sticky: true
+toc_label: Known Issues and Workarounds
+---
+
+
+## Connection issue between a watch and Device Manager
+
+Refer to [Testing Your App on Galaxy Watch][testing_app_on_watch] for information about developing your application and testing it on your watch.
+
+Even if you follow every step correctly, you may face the following error:
+

```

Comment:
Please keep the image or leave the error message same.
The following error message and explanation is the popup content what user will see, and we cannot change the content.

Reply:
My apologies. I've reverted my changes to include the illustration and unchanged text. Added modified _issues/wearable-tools-tizenstudio.md file to my branch.